xbmc/pvr/addons/PVRClients.h

```diff
@@ -64,9 +64,9 @@ namespace PVR
   };
 
   class CPVRClients : public ADDON::IAddonMgrCallback,
-                      public Observer,
-                      private CThread
+                      public Observer
   {
+  friend class CPVRClient;
```

Comment:
```
  #define g_PVRManager       CPVRManager::GetInstance()
  #define g_PVRChannelGroups g_PVRManager.ChannelGroups()
  #define g_PVRTimers        g_PVRManager.Timers()
  #define g_PVRRecordings    g_PVRManager.Recordings()
  #define g_PVRClients       g_PVRManager.Clients()
```

yes I do when dealing with this. at least fo the time being. those public methods gets called from everywhere